app/models/concerns/smart_filterable.rb

```diff
@@ -3,7 +3,8 @@ module SmartFilterable
 
   class_methods do
     def smart_filter(params)
-      (params || {}).to_h.inject(all) do |results, (key, val)|
+      return if params.empty?
+      params.to_h.inject(all) do |results, (key, val)|
```

Comment:
since we expect smart filter to return everything if we don't supply a filter, this change is not necessary.